### PR TITLE
SAI模块-i.mxrt1050evk

### PR DESCRIPTION
**Pull request by xinyigao**
## 拉取/合并请求描述：(PR description)

[
这段方括号里的内容是您**必须填写并替换掉**的，否则PR不可能被合并。**方括号外面的内容不需要修改，但请仔细阅读。**
The content in this square bracket must be filled in and replaced, otherwise, PR can not be merged. The contents outside square brackets need not be changed, but please read them carefully.

请在这里填写您的PR描述，可以包括以下之一的内容：为什么提交这份PR；解决的问题是什么，你的解决方案是什么；
Please fill in your PR description here, which can include one of the following items: why to submit this PR; what is the problem solved and what is your solution;

并确认并列出已经在什么情况或板卡上进行了测试。
And confirm in which case or board has been tested.
]

以下的内容不应该在提交PR时的message修改，修改下述message，PR会被直接关闭。请在提交PR后，浏览器查看PR并对以下检查项逐项check，没问题后逐条在页面上打钩。
The following content must not be changed in the submitted PR message. Otherwise, the PR will be closed immediately. After submitted PR, please use a web browser to visit PR, and check items one by one, and ticked them if no problem.

### 当前拉取/合并请求的状态 Intent for your PR

必须选择一项 Choose one (Mandatory):

- [ ] 本拉取/合并请求是一个草稿版本 This PR is for a code-review and is intended to get feedback
- [ ] 本拉取/合并请求是一个成熟版本 This PR is mature, and ready to be integrated into the repo

### 代码质量 Code Quality：

我在这个拉取/合并请求中已经考虑了 As part of this pull request, I've considered the following:

- [ ] 已经仔细查看过代码改动的对比 Already check the difference between PR and old code
- [ ] 代码风格正确，包括缩进空格，命名及其他风格 Style guide is adhered to, including spacing, naming and other styles
- [ ] 没有垃圾代码，代码尽量精简，不包含`#if 0`代码，不包含已经被注释了的代码 All redundant code is removed and cleaned up
- [ ] 所有变更均有原因及合理的，并且不会影响到其他软件组件代码或BSP All modifications are justified and not affect other components or BSP
- [ ] 对难懂代码均提供对应的注释 I've commented appropriately where code is tricky
- [ ] 本拉取/合并请求代码是高质量的 Code in this PR is of high quality

### CODE REVIEW

File: bsp/imxrt/libraries/drivers/SConscript [hunk 1]
```diff
@@ -45,6 +45,15 @@ if GetDepend('BSP_USING_LCD'):
 if GetDepend('BSP_USING_ETH'):
     src += ['drv_eth.c']
 
+<<<<<<< HEAD
```
Comment (BernardXiong) [inline bsp/imxrt/libraries/drivers/SConscript:48]: It's conflicting line. Please fix it.

**Review by BernardXiong — CHANGES_REQUESTED** [review]
Please resolve the conflict and add the correct copyright information to the header of source files.

**Review by xinyigao — COMMENTED** [review]
”bsp/imxrt/libraries/drivers/SConscript“has been changed

File: bsp/imxrt/libraries/drivers/drv_sai.c [hunk 2]
```diff
@@ -0,0 +1,578 @@
+/*
+ * File: drv_sound.c
+ *
+ * COPYRIGHT (C) 2012-2019, Shanghai Real-Thread Technology Co., Ltd
```
Comment (BernardXiong) [inline bsp/imxrt/libraries/drivers/drv_sai.c:4]: Please use correct license info header.

File: bsp/imxrt/libraries/drivers/drv_sai.h [hunk 3]
```diff
@@ -0,0 +1,66 @@
+/*
+ * File: drv_sound.h
+ *
+ * COPYRIGHT (C) 2012-2018, Shanghai Real-Thread Technology Co., Ltd
```
Comment (BernardXiong) [inline bsp/imxrt/libraries/drivers/drv_sai.h:4]: Please use correct license info header.

File: bsp/imxrt/libraries/drivers/drv_sdio.c [hunk 4]
```diff
@@ -0,0 +1,469 @@
...
+    /*
+    case RESP_R5B:
+        fsl_command.responseType = kCARD_ResponseTypeR5b;
+        break;
+    */
+    default:
+        RT_ASSERT(NULL);
+    }
+
+    // command type
```
Comment (BernardXiong) [inline bsp/imxrt/libraries/drivers/drv_sdio.c:212]: Please remove the unnecessary code.

File: bsp/imxrt/libraries/drivers/drv_sdio.c [hunk 5]
```diff
@@ -0,0 +1,469 @@
...
+
+#ifdef DEBUG
+static void log_toggle(int en)
+{
+    enable_log = en;
+}
+FINSH_FUNCTION_EXPORT(log_toggle, toglle log dumple);
+#endif
+
+//static rt_int32_t _mmc_get_card_status(struct rt_mmcsd_host *host)
```
Comment (BernardXiong) [inline bsp/imxrt/libraries/drivers/drv_sdio.c:388]: Please remove the unnecessary code.

**Review by BernardXiong — CHANGES_REQUESTED** [review]
Please pay attention to your code style.

**Pull request merged**